src/open_inwoner/accounts/models.py

```diff
@@ -282,6 +283,12 @@ def is_email_of_contact(self, email):
             or self.contacts_for_approval.filter(email=email).exists()
         )
 
+    def get_plan_contact_new_count(self):
```

Comment:
Would it be better if we put these in the `query.py`?

Reply:
query.py is usually for things about collections of the model, while this is instance specific so can be an instance method